llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -6043,8 +6178,9 @@ struct VarArgGenericHelper : public VarArgHelperBase {
   void finalizeInstrumentation() override {
     assert(!VAArgSize && !VAArgTLSCopy &&
            "finalizeInstrumentation called twice");
+
     IRBuilder<> IRB(MSV.FnPrologueEnd);
-    VAArgSize = IRB.CreateLoad(IRB.getInt64Ty(), MS.VAArgOverflowSizeTLS);
```

Comment:
can we change runtime to uptr?

Reply:
This comment is also unclear, could you specify your question please?